spec/amp-cache-guidelines.md

```diff
@@ -88,6 +88,10 @@ Accepted MIME types for fonts include the following prefixes:
 |  binary/octet-stream   |
 |  text/plain (not recommended)   |
 
+## Guidelines: Resource size limits
+
+For resources (i.e., HTML, images, fonts) that are larger than 12 MB, do not fetch the resource and return a 404 error.
```

Comment:
This should be:

`For resources (i.e., HTML, images, fonts) that are larger than 12 MB, you may not fetch the resource and return a 404 error.`